docs/data/joy/components/accordion/AccordionTransition.js

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import Box from '@mui/joy/Box';
+import AccordionGroup from '@mui/joy/AccordionGroup';
+import Accordion from '@mui/joy/Accordion';
+import AccordionDetails from '@mui/joy/AccordionDetails';
+import AccordionSummary from '@mui/joy/AccordionSummary';
+import RadioGroup from '@mui/joy/RadioGroup';
+import Radio from '@mui/joy/Radio';
+import Stack from '@mui/joy/Stack';
+import Typography from '@mui/joy/Typography';
+import HighlightedCode from 'docs/src/modules/components/HighlightedCode';
+import BrandingProvider from 'docs/src/BrandingProvider';
+
+export default function AccordionTransition() {
+  const [transition, setTransition] = React.useState('0.4s ease');
+  return (
+    <Stack alignItems="center" spacing={2} sx={{ flex: 1 }}>
+      <RadioGroup
+        orientation="horizontal"
+        value={transition}
+        onChange={(event) => {
+          setTransition(event.target.value);
+        }}
+      >
+        <Radio value="0.3s linear" label="Linear" />
+        <Radio value="0.4s ease" label="Easing" />
+        <Radio value="0.8s cubic-bezier(0.68, -0.6, 0.32, 1.6)" label="Bouncy" />
```

Comment:
It takes 277ms for the accordion to start moving, I don't think that we can't have this option, per https://web.dev/fid/#what-is-a-good-fid-score we are close to "Poor". For comparison, with Easing, I measure this at 27ms.

```suggestion
```

Instead, I think that we could have a small bounce at the end of the animation like https://dribbble.com/shots/20711653-Side-panel-framework-for-Opkey-s-no-code-automation-tool?